src/extension.ts

```diff
@@ -290,6 +291,42 @@ export function activate(context: vscode.ExtensionContext) {
     }
   });
 
+  const copyContextCommand =  vscode.commands.registerCommand('path-copy.copyrelative', () => {
+		vscode.window.showInformationMessage('Copied File Path');
+		if(vscode.window.activeTextEditor) {
+			currPath = vscode.window.activeTextEditor.document.uri.fsPath
+		}
+	});
+
+	const copyExplorerCommand =  vscode.commands.registerCommand('path-copy.copyexplorerelative', (uri:vscode.Uri) => {
+		currPath = uri.fsPath;
+	});
+	
+	const pasteContextCommand =  vscode.commands.registerTextEditorCommand('path-copy.pasterelative', (editor, edit) => {
+		vscode.window.showInformationMessage('Imported relative file path');
+		if(vscode.window.activeTextEditor){
+		let pArr = vscode.window.activeTextEditor.document.uri.fsPath.toString().split("\\");
+		let cArr = currPath.toString().split("\\");
+		 var finalPath = "";
+		cArr.some(createPath);
+ 
+		function createPath(item: string, index: number ) {
+				if(item==pArr[index]){
+				
+				} else if (item != pArr[index]) {
+					var lent = (pArr.length-1) - index;
+					var fS = ("../").repeat(lent);
+					var eS = cArr.slice(index,cArr.length).join("/");
+					finalPath = fS + eS;
+					return true;
+				}
```

Comment:
Please use meaningful names so that logic is easier to read. Also, use an external module (`path`) to find the path difference. Wrap everything in a `try..catch`